synapse/models/risk.py

```diff
@@ -1022,6 +1038,27 @@ def getModelDefs(self):
 
                 )),
 
+                ('risk:outage:cause:taxonomy', {}, ()),
+                ('risk:outage', {}, (
+
+                    ('name', ('str', {'lower': True, 'onespace': True}), {
+                        'doc': 'A name for the outage event.'}),
+
+                    ('period', ('ival', {}), {
+                        'doc': 'The time period where the outage impacted availability.'}),
+
+                    ('cause', ('risk:outage:cause:taxonomy', {}), {
```

Comment:
Do we want a :type taxonomy for "type  of outage"?

Reply:
I think "type" might be too ambiguous and prone to conflation. There may be a few more taxonomies that we need before `:type`. I think we can hold off for now. :+1: